core/shared/src/test/scala/scalaz/zio/FiberRefSpec.scala

```diff
@@ -0,0 +1,86 @@
+package scalaz.zio
+
+class FiberRefSpec extends BaseCrossPlatformSpec {
+  override def is = "FiberRefSpec".title ^ s2"""
+   Create a new FiberRef with a specified value and check if:
+      `get` returns the current value.                         $e1
+      `get` returns the current value for a child.             $e2
+      `set` updates the current value.                         $e3
+      `set` by a child doesn't update parent's value.          $e4
+      `locally` restores original value.                       $e5
+      `locally` restores parent's value.                       $e6
+      `locally` restores undefined value.                      $e7
```

Comment:
We want to eliminate concept of undefined value.

Reply:
It was test description. Should have been "restores initial value"